{"name": "Логин с неверным логином и паролем", "status": "passed", "start": 1731179775986, "stop": 1731179776759, "uuid": "81a1eba9-a4f7-404d-915a-1aaff6de4bd8", "historyId": "6c420bdf31e92239cbc12d3f53fe8949", "testCaseId": "6c420bdf31e92239cbc12d3f53fe8949", "fullName": "tests.test_02_login_user.TestLoginUser#test_login_incorrect_credentials", "labels": [{"name": "story", "value": "2. Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_02_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "MBP-Vlad"}, {"name": "thread", "value": "8112-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_02_login_user"}]}